src/client/models/queries.tsx

```diff
@@ -28,6 +29,19 @@ export const BROKER_METRICS_QUERY = gql`
     }
   }
 `;
+
+export const BROKER_PAGE_QUERY = gql`
```

Comment:
See the broker page comments, but I believe this won't have the functionality that matches the page. Open to talking through what changes need to be made here.